resources/web/wwi/Makefile

```diff
@@ -34,9 +34,31 @@ JS_SOURCES = \
   animation.js \
   webots.js
 
-release debug profile update: $(JS_SOURCES)
-	@echo "# Compressing webots.js.min"
-	@terser $(JS_SOURCES) -o webots.min.js
+NPM_PACKAGES = \
+  babel-minify \
+  @babel/core \
+  @babel/cli \
+  @babel/preset-env
+
+NPM_EXISTS = $(shell which npm 2> /dev/null)
+
+ifeq ($(NPM_EXISTS),)
+release debug profile update:
+	@echo "# Please install nodejs to build 'webots.min.js' using the instructions on the GitHub wiki."
```

Comment:
> We should update the wiki pages about Webots installation regarding the optional packages (node.js) and/or update the scripts (e.g., for example on Windows, msys64_installer.sh should contain mingw-w64-x86_64-nodejs as an optional package.

You're right. I just simplified instructions, added this dependency in the windows installation script, and added the following Wiki sections (difficult to do better because we don't have branches on the wiki):

- https://github.com/omichel/webots/wiki/Linux-Optional-Dependencies#nodejs
- https://github.com/omichel/webots/wiki/macOS-Optional-Dependencies#nodejs